test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1414,8 +1414,9 @@ def revert_mom(self, mom, force=False):
         self.server.expect(NODE, a, id=mom.shortname, interval=1)
         exp_attrib = {'resources_available.ncpus': (GE, 1)}
         counter = self.server.counter(NODE, exp_attrib)
```

Comment:
I know I asked this before, but I am still concerned about it.  In a multi-mom setup, won't this pass if a different mom is up?  How can self.server.counter() know which mom we are talking about?  It's part of the server object, not the mom.

Reply:
I agree with you @bhroam so I have reverted my changes to my initial change of just checking the state of the node to be free.